android/app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <domain-config cleartextTrafficPermitted="true">
+        <domain includeSubdomains="true">10.0.2.2</domain>
```

Comment:
We can't ship that file in the production build with a hardcoded IP address ;) 
It must be: 
+ per dev env (template)
+ tied to the development build only.